.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "singleQuote": true,
+  "semi": false,
+  "trailingComma": "es5",
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "tabSize": 2
+}
```

Comment:
Note that we have the prettier rules under .eslintrc file with the prettier extension so we don´t nee this file

Reply:
My VS dont read the rules there and break all the quotes in the code. I saw too @tiago on #ts-support will added that file.

Have some another way make my prettier of VSCode with the .eslintrc? i not waste some time on that, so i get the fast way to work.

Reply:
Will rename to same name of #738 so we get auto merge on that.